src/haz3lweb/Editors.re

```diff
@@ -63,6 +63,9 @@ let put_editor_and_id = (id: Id.t, ed: Editor.t, eds: t): t =>
 
 let get_zipper = (editors: t): Zipper.t => get_editor(editors).state.zipper;
 
+/* Generate UExp and DHExp on the given editors
```

Comment:
I would explain this as "each mode requires evaluating some number of expressions that are spliced together from the editors in that mode. each expression is given a key for later lookup by the mode"